develop/dev-guide-optimize-sql-overview.md

```diff
@@ -1,31 +1,31 @@
 ---
 title: Optimize SQL Performance Overview
-summary: Provides an overview of SQL performance tuning for TiDB.
+summary: Provides an overview of SQL performance tuning for TiDB application developers.
 ---
 
 # Optimize SQL Performance Overview
 
-This section introduce how to optimize the performance of SQL in TiDB. To get good performance, you can start with the following:
+This document introduces how to optimize the performance of SQL statements in TiDB. To get good performance, you can start with the following aspects:
 
-* SQL Performance Tuning
-* Schema design: Depending on your SQL schema and the data access patterns of your workload, you may need to make some changes to the table's schema in order to avoid transaction contention or hot spots.
+* SQL performance tuning
+* Schema design: Based on your application workload patterns, you might need to make some changes to the table schema to avoid transaction contention or hot spots.
```

Comment:
```suggestion
* Schema design: Based on your application workload patterns, you might need to change the table schema to avoid transaction contention or hot spots.
```